geom/FieldBinaryOperatorTemplate.H

```diff
@@ -0,0 +1,70 @@
+#define _NAME field
+#include "function_overload.macro"
+
+   subroutine _SUB(field_out,field_a,field_b,rc)
+      type(ESMF_Field), intent(inout) :: field_out
+      type(ESMF_Field), intent(inout) :: field_a
+      type(ESMF_Field), intent(inout) :: field_b
```

Comment:
Don't change this for now, but I think we want shorter names for these arguments.   Will make the procedure a bit easier to read, and helps with keywords.  Indicating the type in the name is unnecessary.

Probably  `C, A, B`  as in  `C = A + B`.   But I admit in subroutine form it's less obvious what the right choice is.  Hence - no change yet.

Similar comment for unary case.

Reply:
ok, we can change as needed in a future PR, I don't think anyone but me will be making use if this in the near term so I have no problem going and changing in a future PR.